ngx-tools/type-guards/README.md

```diff
@@ -288,3 +288,17 @@ isUndefined(undefined) // Returns: true
 isUndefined(null)      // Returns: false
 isUndefined('foo')     // Returns: false
 ```
+
+## `isValidDate`
+
+[[source]](src/is-valid-date/is-valid-date.ts)
+
+Determine if an input is a valid date.
+
+```typescript
+import {isValidDate} from '@terminus/ngx-tools/type-guards';
```

Comment:
We should probably add spaces inside these brackets so it matches the library style